__tests__/app.test.js

```diff
@@ -209,6 +209,30 @@ describe('/api/articles endpoint', () => {
         })
       })
   })
+  test("GET: 200 returned articles should be filtered by the specified topic in the supplied query", () => {
+    return request(app)
+    .get('/api/articles?topic=mitch')
+    .expect(200)
+    .then(({body}) => {
+      expect(body.articles).toHaveLength(12)
+      body.articles.forEach((article) => {
+        expect(article.topic).toEqual('mitch')
+      })
+    })
+  })
+  test("GET 204 when supplied topic that has no articles associated with it", () => {
```

Comment:
I would expect this to be a 200 as its a valid query with no rows, same as an article with no comments.